distribution/pom.xml

```diff
@@ -0,0 +1,140 @@
+<!--
+
+    Copyright DataStax, Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.datastax.oss</groupId>
+    <artifactId>java-driver-parent</artifactId>
+    <version>4.0.0-alpha4-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>java-driver-distribution</artifactId>
+  <!-- Should be pom but Javadoc generation requires a "classpath-capable" package -->
+  <packaging>jar</packaging>
+
+  <name>DataStax Java driver for Apache Cassandra(R) - binary distribution</name>
+
+  <!--
+    These dependencies are only here to ensure proper build order and proper inclusion of binaries
+    in the final tarball
+  -->
+  <dependencies>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>java-driver-core</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>java-driver-query-builder</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
should we also include `java-driver-core-shaded`?  I noticed we currently don't include the shaded jar for 3.x's distributable, so I think it's fine to also not include it here.   If the user needs (which is more an exception) they can grab it off maven.

Reply:
Since nobody asked for it on 3.x, leaving it out for now for simplicity.